Source/Core/Core/ConfigManager.cpp

```diff
@@ -405,6 +406,7 @@ void SConfig::LoadGeneralSettings(IniFile& ini)
   general->Get("DumpPath", &m_DumpPath);
   CreateDumpPath(m_DumpPath);
   general->Get("WirelessMac", &m_WirelessMac);
+  general->Get("WiiSDCardPath", &m_strWiiSDCardPath, File::GetUserPath(F_WIISDCARD_IDX));
```

Comment:
Should probably be using File::SetUserPath instead? Just like we do for NANDRootPath and possibly others?